### PR TITLE
bazel: add page

### LINKED ISSUE #7586: page request: bazel
Bazel is a tool that automates software builds and tests.

https://bazel.build/

It doesn't look like there are any man pages (beside https://github.com/bazelbuild/bazel/issues/10519), or great examples of tldr-eqsue content.

### PR DESCRIPTION
**Pull request by saikatsgupta**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Closes #7586

### CODE REVIEW

File: pages/common/bazel.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# bazel
+
+> Open-source build and test tool similar to Make, Maven, and Gradle.
+> More information: <https://bazel.build/reference/command-line-reference>.
+
+- Show available commands and usage:
```
Comment (kbdharun) [inline pages/common/bazel.md:6]: Can you move this `help` command to last?

File: pages/common/bazel.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`bazel help`
+
+- Display runtime info about the bazel server:
+
+`bazel info`
+
+- Print version information for bazel:
```
Comment (kbdharun) [inline pages/common/bazel.md:14]: Same here too, You can place it in last above `help` command.

File: pages/common/bazel.md (hunk 1 above)
Comment (saikatsgupta) [reply pages/common/bazel.md]: Done.

File: pages/common/bazel.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Stop the bazel server:
+
+`bazel shutdown`
+
+- Display runtime info about the bazel server:
+
+`bazel info`
+
+- Show available commands and usage:
```
Comment (CleanMachine1) [inline pages/common/bazel.md:26]:
```suggestion
- Display help:
```

File: pages/common/bazel.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# bazel
+
+> Open-source build and test tool similar to Make, Maven, and Gradle.
+> More information: <https://bazel.build/reference/command-line-reference>.
+
+- Print version information for bazel:
+
+`bazel version`
```
Comment (CleanMachine1) [inline pages/common/bazel.md:8]: Please move this to the bottom and change the description to be

```
- Display version:
```

**Comment by saikatsgupta** [comment]
Not sure why @navarroaxel got removed from reviewers list when I re-requested review from @CleanMachine1 😭

**Comment by kbdharun** [comment]
> Not sure why @navarroaxel got removed from reviewers list when I re-requested review from @CleanMachine1 😭

I will request his review. I think GitHub only allows 2 reviews at a time on the contributor side. Whereas, Collaborators can request up to 15 reviews. 

> I am not sure about this.

**Review by kbdharun — APPROVED** [review]

File: pages/common/bazel.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# bazel
+
+> Open-source build and test tool similar to Make, Maven, and Gradle.
+> More information: <https://bazel.build/reference/command-line-reference>.
+
+- Build the specified targets in the workspace:
```
Comment (navarroaxel) [inline pages/common/bazel.md:6]: which targets? :thinking:

File: pages/common/bazel.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`bazel build`
+
+- Remove output files and optionally stop the server:
```
Comment (navarroaxel) [inline pages/common/bazel.md:10]: Perhaps this better
```suggestion
- Remove output files and stop the server if running:
```

File: pages/common/bazel.md (hunk 5 above)
Comment (saikatsgupta) [reply pages/common/bazel.md]: Good spot, updated command.

**Review by pixelcmtd — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**